packages/common/src/store/account/sagas.ts

```diff
@@ -222,10 +224,23 @@ export function* fetchAccountAsync({
     return
   }
 
-  const accountData = (yield* call([queryClient, queryClient.fetchQuery], {
-    queryKey: getWalletAccountQueryKey(wallet),
-    queryFn: async () => getWalletAccountQueryFn(wallet!, sdk)
-  })) as AccountUserMetadata | undefined
+  let accountData: AccountUserMetadata | undefined
+  try {
+    accountData = (yield* call([queryClient, queryClient.fetchQuery], {
+      queryKey: getWalletAccountQueryKey(wallet),
+      queryFn: async () => getWalletAccountQueryFn(wallet!, sdk),
+      staleTime: Infinity,
+      gcTime: Infinity
+    })) as AccountUserMetadata | undefined
+  } catch (e) {
+    yield* put(resetAccount())
```

Comment:
I think we can simply do nothing in this case and let the `!accountData` check handle. Alternatively, we can remove the later check?